specification/reservations/resource-manager/Microsoft.Capacity/preview/2019-07-19/quota.json

```diff
@@ -87,11 +87,11 @@
       },
       "put": {
         "tags": [
-          "QuotaRequests"
+          "QuotaRequest"
         ],
         "summary": "Submits a Quota Request for a resource provider at the specified location for the specific resource in the parameter.",
         "description": "Submits Quota change request for a resource provider for the specified location for the specific resource in the parameter. To use, first make a Get request to get quota information for the specific resource. This information consists of information regarding that specific resources. For the specific resource, if it requires an update to the quota, update the limit field in the response from the Get request to the new value of quota. Then, submit this updated JSON object to this quota request API. This will update the quota to the value specified. The location header in the response will be used to track the status of the quota request. Please check the provisioningState field in the response.",
-        "operationId": "QuotaRequest_Create",
+        "operationId": "Quota_Create",
```

Comment:
>Quota_Create [](start = 24, length = 12)

_CreateOrUpdate

Reply:
The Patch is for Update. Quota_Update.
Put - Quota_Create

Does it sound fine?

Breaking Changes:
There are no active users.
We are currently in Private preview, so only internal users. I will coordinate with them, after update.

There are no users for autoQuotaIncrease, so making breaking changes in data model.

Thanks.

Reply:
Customers can update using PUT as well, if there are writable properties.  please see other specs ..this is the convention used.

---
In reply to: [413977335](https://github.com/Azure/azure-rest-api-specs/pull/9159#discussion_r413977335) [](ancestors = 413977335)

Reply:
The Patch is for Update. Quota_Update.
Put - Quota_Create

There is no active user.
We are currently in Private preview, so only internal users. I will coordinate with them, after update.

There are no users for autoQuotaIncrease, so making breaking changes in data model.

Thanks.

Reply:
That's right.
Should I apply same opeartionId - Quota_CreateOrUpdate to both Put and patch?
Please suggest.